### PR TITLE
[INLONG-3317][Agent] Change the default heartbeat and report interval to 10s

### LINKED ISSUE #3317: [Improve] Change agent heartbeat/report interval to 10s
### Description

_No response_

### InLong Component

InLong Agent

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kipshi**
### Title Name: [INLONG-3317] Change agent heartbeat/report interval to 10s

where *XYZ* should be replaced by the actual issue number.

Fixes #3317 

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/task/TaskPositionManager.java [hunk 1]
```diff
@@ -91,13 +97,15 @@ private Runnable taskPositionFlushThread() {
...
-                                + "might be deleted by standalone mode, now delete job position in memory", jobId);
+                                    + "might be deleted by standalone mode, now delete job position in memory", jobId);
                             deleteJobPosition(jobId);
                             continue;
                         }
                         flushJobProfile(jobId, jobProfile);
                     }
-                    TimeUnit.SECONDS.sleep(DEFAULT_FLUSH_TIMEOUT);
+                    int flushTime = conf.getInt(AGENT_HEARTBEAT_INTERVAL,
+                            DEFAULT_AGENT_FETCHER_INTERVAL);
```
Comment (EMsnap) [inline inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/task/TaskPositionManager.java:107]: Here the fetcher timeout may be inconsistant with the code

**Review by EMsnap — COMMENTED** [review]
PLAL

**Review by healchow — APPROVED** [review]
LGTM

File: inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/task/TaskPositionManager.java (hunk 1 above)
Comment (kipshi) [reply inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/task/TaskPositionManager.java:107]: In old cold, HeartBeatManager use the same constant(DEFAULT_FLUSH_TIMEOUT) with TaskPositionManager, so i use HEART_BEAT_INTERVAL instead

Comment (EMsnap) [reply inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/task/TaskPositionManager.java:107]: +1

**Review by EMsnap — APPROVED** [review]
+1

**Pull request merged**